releases/v20.2.0-alpha.2.md

```diff
@@ -93,7 +93,6 @@ $ docker pull cockroachdb/cockroach-unstable:v20.2.0-alpha.2
 - Sequences passed as a string argument into views are now tracked as a dependency. For example, `CREATE VIEW v AS SELECT nextval('s')` will now add a dependency on sequence `s`. [#50103][#50103] {% comment %}doc{% endcomment %}
 - Fixed an error for the `ALTER COLUMN TYPE` statement for situations where the data matches byte for byte but CockroachDB needs to validate the `INT8` -> `INT4` conversion. [#50278][#50278]
 - Previously, `infinity` evaluated to a negative value, `-292277022365-05-08T08:17:07Z`. Now `infinity` is the maximum supported timestamp in Postgres that is not infinity. Likewise, `-infinity` is the smallest supported value. Note that this work-in-progress feature currently does not behave exactly like `infinity` in Postgres. [#50311][#50311] {% comment %}doc{% endcomment %}
-- The `CHAR` column [`type`](../v20.2/data-types.html) will now truncate the trailing space characters in line with Postgres. Existing stored `CHAR` entries with spaces at the end of the `CHAR` column [`type`](../v20.2/data-types.html) will no longer return rows with trailing space characters. Use the `LIKE` query to find and modify these rows. [#50492][#50492] {% comment %}doc{% endcomment %}
```

Comment:
The GA release notes of `20.2` don't mention this change at all. Shouldn't it be added there?